lib/network/modules/components/Edge.js

```diff
@@ -144,6 +144,23 @@ class Edge {
     // only deep extend the items in the field array. These do not have shorthand.
     util.selectiveDeepExtend(fields, parentOptions, newOptions, allowDeletion);
 
+    // Only use endPointOffset values (from and to) if it's valid values
+    if (newOptions.endPointOffset !== undefined) {
```

Comment:
This shouldn't report values that are not there as invalid, i.e. `endPointOffset: { from: 7 }` shouldn't be reported.

Reply:
You have to define both from and to values.

Reply:
I change it so that you can define just one endpoint on a edge.